build.gradle.kts

```
@@ -94,7 +93,6 @@ dependencies {
 }
 
 java {
-    withJavadocJar()
     withSourcesJar()
```

```suggestion
    withJavadocJar()
    withSourcesJar()
```

As @vlsi pointed our, it could be problematic to have `-javadoc` jar completely missing. Restoring that one line generates "empty" java file, which should be enough for Maven Central requirements (once we will decide to publish also there in the future).

---

I agree, publishing an empty javadoc jar would be reasonable workaround